examples/csharp/HyperspaceApp/.gitignore

```diff
@@ -0,0 +1,369 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
can we merge this .gitignore file to <root>/.gitignore ? Unless it's required here?

Reply:
Hmm... the rules in this file are all specific to Visual Studio / .NET so I'm wondering if we should keep them separate?

Reply:
I am fine leaving these separate (maybe under csharp)?

Reply:
Moved to `csharp` folder.